k2/csrc/properties.cc

```diff
@@ -55,6 +55,22 @@ bool IsTopSorted(const Fsa &fsa) {
   return true;
 }
 
+bool IsArcSorted(const Fsa &fsa) {
+  ArcLabelCompare comp;
```

Comment:
Check that the fsa is not empty.

Reply:
it will work on empty fsa 
https://github.com/danpovey/k2/blob/aac136e0a90c54e0bd5a2400ecc1466b5115bbe2/k2/csrc/properties.cc#L64

also see tests
https://github.com/danpovey/k2/blob/aac136e0a90c54e0bd5a2400ecc1466b5115bbe2/k2/csrc/properties_test.cc#L189-L192